tests/types/types.go

```diff
@@ -172,31 +172,6 @@ func GetBaseDisk() []Disk {
 					Length:         262144,
 					FilesystemType: "vfat",
 					Hybrid:         true,
-					Files: []File{
```

Comment:
If we're going to remove these from the baseline checking can we add these files as validation in the `In` & `Out` partitions of the `EmptyUserdata` test (or a similar test) to make sure that we don't introduce a regression where files are removed that shouldn't be?